doc/source/user_guide/visualization.rst

```diff
@@ -444,8 +444,7 @@ keyword, will affect the output type as well:
 
 ================ ======= ==========================
 ``return_type=`` Faceted Output type
```

Comment:
I think we can remove the `=` - I know it was there originally, but I don't think it really makes sense